### PR TITLE
Changes Reflecting BAG3 Open Sourcing Updates

### PR DESCRIPTION
**Pull request by FeliciaG**
Changes have been made to reflect updates used in June 2022 and December 2022 tapeouts. This update is compatible with the newly released BAG3++, which is has a completely publicly available codebase.

### CODE REVIEW

**Review by mithro — COMMENTED** [review]
Could you fix the copyright header removals / changes?

**Comment by FeliciaG** [comment]
The copyright headers should be fixed now, please let me know if there are other issues.

File: OA/BAG_prim/cdsinfo.tag [hunk 1]
```diff
@@ -1,13 +1,17 @@
 # Copyright 2019-2021 SkyWater PDK Authors
```
Comment (mithro) [inline OA/BAG_prim/cdsinfo.tag]: This file is still be weirdly modified.

File: README.md [hunk 2]
```diff
@@ -1,30 +1,66 @@
-# BAG (BAG AMS Generator) Primitives Library for [SKY130](https://github.com/google/skywater-pdk)
```
Comment (mithro) [inline README.md:1]: Could we keep this header?

File: README.md [hunk 3]
```diff
@@ -1,30 +1,66 @@
...
+# skywater130
 
-This repository contains the required primitives to use the BAG3 framework with
```
Comment (mithro) [inline README.md:3]: Could we keep this description?

File: install.sh [hunk 4]
```diff
@@ -31,19 +33,14 @@ cp_files=( ".cdsenv.personal"
 ln_files=( "bag_config.yaml"
            ".cdsenv"
            ".cdsinit"
-	   ".simrc"
+	         ".simrc"
```
Comment (mithro) [inline install.sh]: Random unneeded change?

File: install.sh [hunk 5]
```diff
@@ -18,6 +19,7 @@
 # https://github.com/google/skywater-pdk-libs-sky130_bag3_pr/blob/master/LICENSE.alternative
 #
 # SPDX-License-Identifier: BSD-3-Clause OR Apache 2.0
+#!/usr/bin/env bash
```
Comment (mithro) [inline install.sh]: This line seems wrong?

File: install.sh [hunk 6]
```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env bash
+#!/bin/bash
```
Comment (mithro) [inline install.sh]: Can we keep the hashbang as previous?

**Review by mithro — CHANGES_REQUESTED** [review]
Getting pretty close. There are still some fixes needed.

 * The README.md needs to keep the old content. Your new additions are fine though.
 * A few files have been slightly mangled when fixing the copyright header.
 * Duplicate `# -*- coding: utf-8 -*-` lines (needs to be the second line in the file).
 * Some files still missing copyright headers.

**Comment by FeliciaG** [comment]
Sorry for the slow turnaround. I fixed: 

- the README to include the old content
- mangled copyright headers
- Duplicated `# -*- coding: utf-8 -*- ` lines
- missing copyright headers (I think I hit all of them, but if you catch more missing ones, please let me know)

**Comment by mithro** [comment]
@FeliciaG - I have merged this pull request but there are a couple of issues I identified, see
 * #9 
 * #10 
 * #11

**Comment by mithro** [comment]
@FeliciaG - I also invited you to this repository so that these issues can be assigned to you.

**Pull request merged**